lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -201,8 +208,9 @@ def self.payload_to_melt_format(data)
         # To save on unnecessary data transmission, trim the entity.type
         # sent by classic logs-in-context
         common_attributes.delete(ENTITY_TYPE_KEY)
-
-        common_attributes.merge!(NewRelic::Agent.agent.log_event_aggregator.attributes.custom_attributes)
+        aggregator = NewRelic::Agent.agent.log_event_aggregator
+        common_attributes.merge!(aggregator.attributes.custom_attributes)
+        common_attributes.merge!(aggregator.labels)
```

Comment:
This leaves me wondering if we should leverage the LogEventAttributes class for the labels work too, rather than leaving all the labels work inside the aggregator. However, this may make labels more difficult to remove when the aggregator is reset. Thoughts?